jplag.endToEndTesting/pom.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>de.jplag</groupId>
+		<artifactId>aggregator</artifactId>
+		<version>${revision}</version>
+	</parent>
+	<artifactId>jplag.endToEndTesting</artifactId>
+
+	<dependencies>
+		<dependency>
+			<groupId>de.jplag</groupId>
+			<artifactId>jplag</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>junit</groupId>
+			<artifactId>junit</artifactId>
+			<version>4.9</version>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.json</groupId>
+			<artifactId>json</artifactId>
+			<version>20070829</version>
+		</dependency>
```

Comment:
please use jackson-json if possible

Reply:
I will take a look and then adjust. But this will take some time, because it requires changes in some places.

Reply:
Very good tip thank you! made the code much cleaner and more readable. 
done